pkg/sdk/applications_validations_gen.go

```diff
@@ -0,0 +1,77 @@
+package sdk
+
+var (
+	_ validatable = new(CreateApplicationOptions)
+	_ validatable = new(DropApplicationOptions)
+	_ validatable = new(AlterApplicationOptions)
+	_ validatable = new(ShowApplicationOptions)
+	_ validatable = new(DescribeApplicationOptions)
+)
+
+func (opts *CreateApplicationOptions) validate() error {
```

Comment:
according to the docs DEBUG cannot be set if not using the `USING` syntax (first one here: https://docs.snowflake.com/en/sql-reference/sql/create-application#syntax). Is this the error in the docs or should it be checked?

Reply:
added validation